### ISSUE TITLE
[apps] Convert macro to lambda and fix the call-sites

### ISSUE THREAD

**Issue by kunaltyagi** [PointCloudLibrary/pcl#3897]
Convert macro to lambda and fix the call-sites.

Locations:
*  apps/include/pcl/apps/manual_registration.h
*  apps/include/pcl/apps/openni_passthrough.h
* apps/include/pcl/apps/pcd_video_player.h
* apps/include/pcl/apps/timer.h

_Originally posted by @kunaltyagi in https://github.com/PointCloudLibrary/pcl/pull/3885_

**Comment by SunBlack** [comment]
There are much more of it:

* pcl\apps\include\pcl\apps\manual_registration.h
* pcl\apps\include\pcl\apps\openni_passthrough.h
* pcl\apps\include\pcl\apps\organized_segmentation_demo.h
* pcl\apps\include\pcl\apps\pcd_video_player.h
* pcl\apps\include\pcl\apps\timer.h
* pcl\apps\src\openni_3d_concave_hull.cpp
* pcl\apps\src\openni_3d_convex_hull.cpp
* pcl\apps\src\openni_boundary_estimation.cpp
* pcl\apps\src\openni_color_filter.cpp
* pcl\apps\src\openni_fast_mesh.cpp
* pcl\apps\src\openni_feature_persistence.cpp
* pcl\apps\src\openni_grab_frame.cpp
* pcl\apps\src\openni_ii_normal_estimation.cpp
* pcl\apps\src\openni_klt.cpp
* pcl\apps\src\openni_mls_smoothing.cpp
* pcl\apps\src\openni_octree_compression.cpp
* pcl\apps\src\openni_organized_compression.cpp
* pcl\apps\src\openni_uniform_sampling.cpp
* pcl\apps\src\openni_voxel_grid.cpp

Just care: Not all are completely equal

**Comment by chaytanyasinha** [comment]
@kunaltyagi May I do this?

**Comment by SunBlack** [comment]
After #3885 is merged - yes ;-)

**Comment by chaytanyasinha** [comment]
@SunBlack I didn't knew you were already working on it.

**Comment by SunBlack** [comment]
No, I'm not working on it. I'm just applying clang-format to apps and during this we detected this code duplication. But it makes no sense to work on it, as long the formatting is not, as you would get a big merge conflict.

**Comment by chaytanyasinha** [comment]
@SunBlack Okay.

**Comment by kunaltyagi** [comment]
#3885 is now merged, and this issue is open for action

**Comment by jmaiolini** [comment]
Hi guys @kunaltyagi @SunBlack, can I take this issue if it is still relevant? 
In case, do you want me to define the lambda every time it is called? What do you mean to fix the call-sites? Or should I declare it as it is callable multiple times?
Can I assume the list above contains all the files to modify?

**Comment by kunaltyagi** [comment]
1. Convert macro into a lambda. Might require adding some parameter compared to the macro version(this is called fixing the call site)
2. Find files with similar lambda, convert that into a function in a header file

**Comment by jmaiolini** [comment]
Just to be 100% sure, can you give me more details about point 2? Did you mean files with `FPS_CALC` ?
